mmdet/visualization/local_visualizer.py

```diff
@@ -399,3 +401,201 @@ def add_datasample(
             mmcv.imwrite(drawn_img[..., ::-1], out_file)
         else:
             self.add_image(name, drawn_img, step)
+
+
+def random_color(seed):
+    """Random a color according to the input seed."""
+    np.random.seed(seed)
+    colors = sns.color_palette()
+    color = colors[np.random.choice(range(len(colors)))]
+    color = tuple([int(255 * c) for c in color])
+    return color
+
+
+@VISUALIZERS.register_module()
+class TrackLocalVisualizer(Visualizer):
```

Comment:
Could this class inherit from  `DetLocalVisualizer` and simplify these code? Because the that the `DataSample` passed into `add_datasample' is still an instance of  `DetDataSample`. Or, even just modify the DetLocalVisualizer since MOT/VIS only adds instances id property?